.ci/openshift-ci/common.sh

```diff
@@ -555,12 +555,12 @@ testCloneGitRepoNoProjectExists() {
     runTestWorkspaceWithGitRepoUrl ${WS_NAME} ${PROJECT_NAME} ${GIT_REPO_URL} ${OCP_USER_NAMESPACE}
     echo "------- [INFO] Check the private repository is NOT cloned with NO PAT/OAuth setup. -------"
     testProjectIsCloned ${PROJECT_NAME} ${OCP_USER_NAMESPACE} && \
-    { echo "####### [ERROR] Project file /projects/${PROJECT_NAME}/${YAML_FILE_NAME} should NOT be present. #######
+    { echo "####### [ERROR] Project file /projects/${PROJECT_NAME}/${TEST_FILE_NAME} should NOT be present. #######
 ####### Cause possible: PR code regress or service is changed. Need to investigate it. #######" && exit 1; }
-    echo "======= [INFO] Project file /projects/${PROJECT_NAME}/${YAML_FILE_NAME} is NOT present. This is EXPECTED. ======="
+    echo "======= [INFO] Project file /projects/${PROJECT_NAME}/${TEST_FILE_NAME} is NOT present. This is EXPECTED. ======="
 }
 
-# Test that the repository is cloned when PAT, OAuth or SSH is configured
+# Test that the repository is cloned when a public repositoty, PAT, OAuth or SSH is configured
```

Comment:
It's not clear what `... the repository is cloned when a public repository.... is configured` means.

Reply:
The annotation is fixed.